modules/functions-list-kn.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies
+
+// * /serverless/cli_reference/kn-func-ref.adoc
+
+[id="functions-list-kn_{context}"]
+= Listing existing functions
+
+You can list existing functions by using `kn func list`. If you want to list functions that have been deployed as Knative services, you can also use `kn service list`.
+
+[NOTE]
+====
+If the namespace is not specified for the following commands, this defaults to the current namespace.
+====
+
+.Procedure
+
+. List existing functions:
++
+[source,terminal]
+----
+$ kn func list [-n <namespace> -p <path>]
+----
++
+.Example output
+[source,terminal]
+----
+NAME           NAMESPACE  RUNTIME  URL                                                                                      READY
+example-function  default    node     http://example-function.default.apps.ci-ln-g9f36hb-d5d6b.origin-ci-int-aws.dev.rhcloud.com  True
+
+----
+. List functions deployed as Knative services:
++
+[source,terminal]
+----
+$ kn service list -n <namespace>
+----
++
+.Example output
+[source,terminal]
+----
+NAME            URL                                                                                       LATEST                AGE   CONDITIONS   READY   REASON
+example-function   http://example-function-default.apps.ci-ln-g9f36hb-d5d6b.origin-ci-int-aws.dev.rhcloud.com   example-function-gzl4c   16m   3 OK / 3     True
```

Comment:
Here just minor change in the route it will generate (to match with the one from kn func list command)

```suggestion
example-function   http://example-function.default.apps.ci-ln-g9f36hb-d5d6b.origin-ci-int-aws.dev.rhcloud.com   example-function-gzl4c   16m   3 OK / 3     True
```